plutus-metatheory/src/Utils.lagda

```diff
@@ -202,14 +202,20 @@ a kind called `#` which is used for sized integers and bytestrings.
 \begin{code}
 data Kind : Set where
   *   : Kind               -- type
+  ♯   : Kind               -- builtin
   _⇒_ : Kind → Kind → Kind -- function kind
 
-{-# FOREIGN GHC import PlutusCore                       #-}
-{-# FOREIGN GHC {-# LANGUAGE GADTs, PatternSynonyms #-} #-}
-{-# FOREIGN GHC type KIND = Kind ()                     #-}
-{-# FOREIGN GHC pattern Star    = Type ()               #-}
-{-# FOREIGN GHC pattern Arrow k j = KindArrow () k j    #-}
-{-# COMPILE GHC Kind = data KIND (Star | Arrow)         #-}
+-- {-# FOREIGN GHC import PlutusCore                       #-}
+-- {-# FOREIGN GHC {-# LANGUAGE GADTs, PatternSynonyms #-} #-}
+
+-- {-# FOREIGN GHC type KIND = Kind ()                     #-}
+
+-- {-# FOREIGN GHC pattern Star    = Type ()               #-}
+
+-- {- TODO: Map Sharp to the correct kind once it is added to the Haskell implementation -}
+-- {-# FOREIGN GHC pattern Sharp   = Type ()               #-}
+-- {-# FOREIGN GHC pattern Arrow k j = KindArrow () k j    #-}
```

Comment:
Just to be clear, we are not even sure that adding `#` to the implementation is going to happen. Nobody is working on that and from what I know nobody plans to in any foreseeable future. There's plenty of things our users care about and having the builtin kind just isn't one of them.

Reply:
I understand. I can probably remove this comment since we now have code for mapping between Haskell and Agda kinds (`convK` and `unconvK`)

Reply:
Comment Removed